pyccel/ast/core.py

```diff
@@ -2907,8 +2942,11 @@ class ValuedArgument(Basic):
     >>> n
     n=4
     """
+    def __new__(cls, *args, **kwargs):
+        kwargs.pop('kwonly', False)
```

Comment:
Why are you removing `kwonly` from the keyword arguments? Maybe you wanted to write `kwargs.setdefault('kwonly', False)` instead?

Reply:
I am removing it because Basic does not accept kwargs. This is due to me using the Basic.__new__ call incorrectly. I have fixed this now